packages/govuk-frontend/src/govuk/core/_theme.scss

```diff
@@ -0,0 +1,62 @@
+@mixin _govuk-theme-light() {
```

Comment:
**note** There'd be a couple of extra decision around these mixins for an actual implementation:

- public or private?
  
  Don't think we need them to be public unless we plan to support setting themes on only part of the page (for ex. through not root level `.govuk-theme--light` and `.govuk-theme--dark` classes that could be used on any element)
- 'theme' or 'colour-scheme'?
  
  Went with 'theme' to avoid confusion with a mixin that would only control the 'color-scheme' property
- separate mixins or single mixin with argument (`_govuk-theme(light)` or `_govuk-theme(dark)`), possibly pulling from a map (`$govuk-themes: (light: ( brand: $govuk-brand-colour, /* ... */), dark: (/* ... */))`)?
  
  Don't think we need the extra complexity of receiving arguments unless we plan to let users customise parts of the theme (in Sass at least, as they can already override the variables by setting their own values on `:root` anyway).